servant/src/Servant/API/UVerb.hs

```diff
@@ -105,3 +98,27 @@ instance KnownStatus n => HasStatus (WithStatus n a) where
 -- Backwards compatibility is tricky, though: this type alias would mean people would have to
 -- use 'respond' instead of 'pure' or 'return', so all old handlers would have to be rewritten.
 data UVerb (method :: StdMethod) (contentTypes :: [*]) (as :: [*])
+
+instance {-# OVERLAPPING #-} MimeRender JSON a => MimeRender JSON (WithStatus _status a) where
+  mimeRender contentTypeProxy (WithStatus a) = mimeRender contentTypeProxy a
```

Comment:
No need for the overlapping pragma.  `MimeRender` is overlappable already

Reply:
tried without, didn't' work.  (i know, weird, right?  can you try to reproduce it?  perhaps i bonked it somehow.)